assets/css/layout/_header.scss

```diff
@@ -0,0 +1,149 @@
+// Header styles
+
+header {
+    .brand {
+        a {
+            color: $black;
+            font-size: rem(48);
+            font-weight: $font-weight-normal;
+            padding-right: rem(16);
+        }
+
+        svg {
+            margin: rem(-8) rem(16) 0 rem(16);
+            width: rem(48);
+        }
+    }
+
+    .site-nav {
+        align-items: center;
+        background-color: $navy-light;
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        padding: 0 rem(16);
+
+        // Toggle menu button on mobile design
+        // Override the default button styles from _base.scss and apply the new style
+        button {
+            background: none;
+            border: none;
+            border-radius: 0;
+            box-shadow: none;
+            color: $white;
+            font-size: rem(16);
+            padding: rem(8);
+        }
+
+        button::before {
+            content: none;
+        }
+        // End of overriding the default button styles from _base.scss
+
+        button:hover {
+            background-color: $orange-focus;
+        }
+
+        // Styles for the navigation menu
+        nav {
+            background: $navy-dark;
+            display: none;
+            left: 0;
+            position: absolute;
+            top: rem(120);
+            width: 100%;
+
+            a {
+                color: $white;
+                display: block;
+                float: none;
+                font-weight: $font-weight-normal;
+                margin: rem(16) rem(16);
+                text-align: left;
+                white-space: nowrap;
+
+                &:hover,
+                &:focus {
+                    color: $navy-light;
+                }
+            }
+        }
+
+        button[aria-expanded="true"] + nav {
+            display: block;
+        }
+
+        // Styles for the search container
+        .search-container {
+            align-items: center;
+            display: flex;
+
+            input[type="search"] {
+                background-color: white;
+                background-image: url("../../static/img/search.svg");
+                background-position: center;
+                background-repeat: no-repeat;
+                border: 0;
+                border-radius: 50%;
+                outline: none;
+                padding: rem(8) rem(16);
+                width: rem(16);
+
+                &:focus {
+                    background-position: left rem(8) center;
+                    border-radius: rem(28);
+                    padding-left: rem(32);
+                    width: rem(160);
+                }
+            }
+        }
+    }
+}
+
+@media screen and (min-width: 600px) {
+    header {
+        .site-nav {
+            justify-content: space-around;
```

Comment:
I'd suggest setting a `max-width` for this element with `margin-left` and `margin-right` set to auto. On my 27" monitor it looks very strange to have the items spread out this far:

<img width="2543" alt="Screen shot of desktop navigation on large screen." src="https://user-images.githubusercontent.com/605361/77354683-db9eba00-6d08-11ea-9ad5-effcf792a554.png">

Reply:
Sorry @cindyli, I think I explained this badly. The header should still be full width but the nav menu/search contaienr should be set to a max width. Right now there is a weird gap on either side, and I think the blue background should extend to full width.

<img width="2542" alt="Screen Shot 2020-03-25 at 11 13 26 AM" src="https://user-images.githubusercontent.com/605361/77565347-b2f1fe00-6e89-11ea-99da-64c52285f4c9.png">